sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2300,6 +2332,91 @@ def __getitem__(self, key):
     return self.attributes[key].__get__(self, WriteResult)
 
 
+def _default_io_expansion_service(append_args=None):
+  return BeamJarExpansionService(
+      'sdks:java:io:google-cloud-platform:expansion-service:build',
+      append_args=append_args)
+
+
+class StorageWriteToBigQuery(PTransform):
+  """Writes data to BigQuery using Storage API."""
+  URN = "beam:schematransform:org.apache.beam:bigquery_storage_write:v1"
+  FAILED_ROWS = "failed_rows"
+  FAILED_ROWS_WITH_ERRORS = "failed_rows_with_errors"
+
+  def __init__(
+      self,
+      table,
+      create_disposition="",
```

Comment:
Please make sure default values are set to valid values. Many users may try to use the BQ sink without overriding the optional arguments.

Reply:
These defaults are already handled in the Java SchemaTransform implementation (e.g. an empty string for create disposition means no disposition will be explicitly set, so the default will be CREATE_IF_NEEDED). I can include the defaults here too (ie. `create_disposition="CREATE_IF_NEEDED"`) to make it more clear what they are.